build.gradle

```diff
@@ -262,6 +289,16 @@ task codeCoverageReport(type: JacocoReport) {
   }
 }
 
+// Automatically close and release the staging repository that gets created
+// during the upload to maven central.
+nexusStaging {
+  username System.getenv('SDA_SONATYPE_USER')
+  password System.getenv('SDA_SONATYPE_PASSWORD')
+  packageGroup "org.sdase.commons"
+  // wait for max. 10 minutes
+  numberOfRetries 300
```

Comment:
How do number of retries map on minutes? Are 10 minutes enough? I already encountered quite long times of the checks in Sonatype staging with much smaller projects.

Reply:
The delay between the checks is [2 seconds by default](https://github.com/Codearte/gradle-nexus-staging-plugin#configuration). By default it only waited for 40 seconds but that didn't worked since it took about 1-2 Minutes. I guess we will see if it is enough.